src/webgpu/api/validation/queue/writeBuffer.spec.ts

```diff
@@ -132,6 +133,29 @@ Also verifies that the specified data range:
     }
   });
 
+g.test('write_sizes_overflow')
+  .desc(
+    `
+Tests writing large sizes that may overflow a 64-bit integer.
+`
+  )
+  .paramsSubcasesOnly([
+    { size: 16, _valid: true }, // control case
+    { size: 0xfffffffffffff800, _valid: false }, // The largest number less than the maximum of uint64
+    { size: 0xffffffffffffff00, _valid: false }, // The smallest number overflow uint64
```

Comment:
Sorry I wasn't clear about what I meant by "add an offset". These are useful to test WebIDL implementations, but they should already be robust to this kind of overflow. The WebGPU-specific implementation (e.g. Blink or Dawn), however, is likely to do something like `offset + size` and we want to test that the implementation doesn't allow that to overflow. So cases are also needed which set `offset` to a larger value.

Reply:
Thanks for your clarification. 
As we discussed offline, I had filed an issue for the TypeError thrown if a Number larger that MAX_SAFE_INTEGER (https://bugs.chromium.org/p/dawn/issues/detail?id=961), and still expected it throw OperationError in our cases. If it's actually the WebIDL-required behavior, we will remove  the overflow case at that time.

Reply:
We got an answer on the bug, and indeed it turns out it's invalid in WebIDL! Sorry for my misreading of the WebIDL spec. Let's remove the overflow cases then 👍

Reply:
Got it! Removed.